connectors/migrations/20241211_fix_gdrive_parents.ts

```diff
@@ -69,58 +90,53 @@ async function migrate({
     for (const file of googleDriveFiles) {
       const internalId = file.dustFileId;
       const driveFileId = file.driveFileId;
-      const parents = [driveFileId];
-      let current: string | null = file.parentId;
-      while (current) {
-        parents.push(current);
-        current = parentsMap[current] || null;
+      const parents = getParents(file.parentId, driveFileId, parentsMap);
```

Comment:
hard to understand here; getParents take a fileId and a nodeId, the fileId should be the first parent, the nodeId should be the child, and we get the parents for the child but without the child as parents[0] which is added later, is that it? at a minimum, args should be renamed for this to be clear

would it be possible to make a single function getParents(nodeId, parentsMap) that would return the exact parents array we want, and minimize the unshifts we do later?

Reply:
agree the params are unclear, but we can't just use one single nodeId param, because if nodeId is a spreadsheet id it won't be in the parentsMap (which contains only files). The nodeId param is just here for logging.

Reply:
replaced nodeId with logger param